pkg/inspector/image-inspector.go

```diff
@@ -133,12 +142,32 @@ func (i *defaultImageInspector) Inspect() error {
 		} else {
 			i.meta.OpenSCAP.Status = iiapi.StatusSuccess
 		}
+		scanResults.Results = append(scanResults.Results, openscap.ParseResults(scanReport)...)
```

Comment:
@pweil- all we need here is `clamav.ParseResults()` that converts whatever clam provides to the result format.